### ISSUE TITLE
Render elements instead of whole pages

### ISSUE THREAD

**Issue by tuxiqae** [MikeWalrus/docx-you-want#3]
Do you think that it'd be possible to loop through the elements in the PDF in order to create matching elements in the DOCX version without using an image of the page?

**Comment by MikeWalrus** [comment]
I am not sure but isn't this something that MS Word or Adobe Acrobat does when converting PDFs?

**Comment by tuxiqae** [comment]
When converting DOCX to PDF maybe, do they provide a way to convert PDFs into DOCX files?

**Comment by MikeWalrus** [comment]
Yes, I think they both provide such functionality. I remember you'll get editable DOCX files, and the structures are roughly the same as the PDF but look kind of ugly.

**Comment by tuxiqae** [comment]
So maybe it'd be possible to move that functionality outside of those proprietary apps into your independent app.
Maybe it'll even be possible to make it perform better.

**Comment by MikeWalrus** [comment]
Oh, there's [pandoc](https://github.com/jgm/pandoc) as well. It's free software and does a good job. That's a real solution you can rely on, while I consider my `docx-you-want` as some sort of hack you can use once in a while or just for fun. 🤣
